server_admin/topics/admin-cli.adoc

```diff
@@ -1084,7 +1084,7 @@ Use a dedicated [command]`add-roles` command to add realm roles to a user.
 Use the following example to add the [command]`user` role to user [command]`testuser`.
 [options="nowrap"]
 ----
-$ kcadm.sh add-roles --username testuser --rolename user -r demorealm
+$ kcadm.sh add-roles --uusername testuser --rolename user -r demorealm
```

Comment:
This doesn't look right. Surely the param name isn't "--uusername", but "--username" as it is in the docs currently